tests/flow/test_index_create.py

```diff
@@ -1,9 +1,19 @@
 from common import *
+from pathos.pools import ProcessPool as Pool
 
 GRAPH_ID = "index"
 redis_graph = None
 redis_con = None
 
+def issue_query(i):
+    env = Env(decodeResponses=True)
+    redis_con = env.getConnection()
+    for _ in range(1, 100):
+        pipe = redis_con.pipeline()
+        pipe.execute_command("GRAPH.QUERY", f"x{i}", f"CREATE (a:L), (n:L), (n)-[:T]->(a)")
+        pipe.execute_command("GRAPH.QUERY", f"x{i}", f"CREATE INDEX FOR ()-[n:T]-() ON (n.p)")
```

Comment:
Why do we need a formatted string:
`f"CREATE (a:L), (n:L), (n)-[:T]->(a)"`
and
`f"CREATE INDEX FOR ()-[n:T]-() ON (n.p)"`
?

can't we drop `f` from both ?